### PR TITLE
[Extensions] Add publish feature support for extensions

### PR DESCRIPTION
**Pull request by Chadha93**
**Notes for Reviewers**
https://github.com/layer5labs/meshery-extensions/pull/1750

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Chadha93** [comment]
Will merge it along with https://github.com/layer5labs/meshery-extensions/pull/1750

**Review by leecalcote — APPROVED** [review]
LGTM

File: ui/components/NavigatorExtension.js [hunk 1]
```diff
@@ -90,6 +91,7 @@ function NavigatorExtension({ grafana, prometheus, updateLoadTestData, url, isDr
         ConfirmationModal,
         getComponentsinFile,
         UploadImport,
+        PublishModal,
```
Comment (leecalcote) [inline ui/components/NavigatorExtension.js:94]:
```suggestion
        PublishModal,
```
remove whitespace

File: ui/remote-component.config.js [hunk 2]
```diff
@@ -148,6 +148,7 @@ module.exports = {
     "@material-ui/icons/GetApp" : require("@material-ui/icons/GetApp"),
     "@material-ui/core/Avatar" : require("@material-ui/core/Avatar"),
     "@material-ui/lab/AvatarGroup" : require("@material-ui/lab/AvatarGroup"),
-    "@material-ui/icons/Person" : require("@material-ui/icons/Person")
+    "@material-ui/icons/Person" : require("@material-ui/icons/Person"),
+    "@material-ui/icons/Public" : require("@material-ui/icons/Public")
```
Comment (Abhishek-kumar09) [inline ui/remote-component.config.js:152]: Please remove this.

Comment (Chadha93) [reply ui/remote-component.config.js]: Removed.

**Pull request merged**